test_runner/flank.ios.yml

```diff
@@ -126,3 +126,7 @@ flank:
 
   ## The max time this test run can execute before it is cancelled (default: unlimited).
   # run-timeout: 60m
+
+  ## Keeps the full path of downloaded files. Required when file names are not unique.
+  ## Default: false
+  # keep-file-path: false
```

Comment:
![image](https://user-images.githubusercontent.com/1173057/79093488-468c5100-7d09-11ea-8447-fc7895497c08.png)

let's keep the newline at the end of the file
https://unix.stackexchange.com/a/18789